docker-multinode/common.sh

```diff
@@ -191,10 +195,11 @@ kube::multinode::start_k8s_master() {
     --privileged \
     --restart=${RESTART_POLICY} \
     ${KUBELET_MOUNTS} \
-    gcr.io/google_containers/hyperkube-${ARCH}:${K8S_VERSION} \
-    /hyperkube kubelet \
+    ${KUBELET_ENV} \
+    taimir93/hyperkube-amd64:v1.3.0 \
```

Comment:
The modified hyperkube image. Will change it back to the original registry once we settle on a solution.